sdcm/cluster_k8s/__init__.py

```diff
@@ -807,7 +813,22 @@ def deploy_scylla_cluster(self, node_pool: CloudK8sNodePool = None, node_prepare
             affinity_modifiers.extend(node_pool.affinity_modifiers)
             if node_prepare_config:
                 LOGGER.info("Install DaemonSets required by scylla nodes")
-                self.apply_file(node_prepare_config, modifiers=affinity_modifiers, envsubst=False)
+                if self.params.get('k8s_enable_performance_tuning') and self.CREATE_NODE_CONFIG_CR:
```

Comment:
WUT, would it better it have a @property like this:
```
    @property
    def _enable_performance_tuning(self):
        return self.params.get('k8s_enable_performance_tuning') and self.CREATE_NODE_CONFIG_CR
```

Reply:
Sounds ok, will do.

Reply:
done